lib/blocks.php

```diff
@@ -187,6 +194,29 @@ function do_blocks( $content ) {
 	add_filter( 'the_content', 'do_blocks', 7 ); // BEFORE do_shortcode() and oembed.
 }
 
+if ( ! function_exists( '_restore_wpautop_hook' ) ) {
+	/**
+	 * If do_blocks() needs to remove wp_autop() from the `the_content` filter,
```

Comment:
`wp_autop` ~> `wpautop`

Reply:
Thanks for catching that. It's fixed now.